packages/core/src/repository/cache.repository.ts

```diff
@@ -189,6 +189,9 @@ export class CacheRepository<T> implements GetRepository<T>, PutRepository<T>, D
     }
 
     public async deleteAll(query: Query, operation: Operation): Promise<void> {
+        // tslint:disable-next-line:max-line-length
+        console.warn('[DEPRECATION] `deleteAll` will be deprecated. Use `delete` instead.');
```

Comment:
We should use always the logger as the way to write logs. If the class doesn't have, you just need to request the Logger interface in the constructor. 

It looks like a silly thing but it's important to avoid log things in places where we don't want.

Reply:
Makes total sense